nfpm.go

```diff
@@ -303,6 +305,7 @@ func (i *Info) parseSemver() {
 type Overridables struct {
 	Replaces   []string       `yaml:"replaces,omitempty" json:"replaces,omitempty" jsonschema:"title=replaces directive,example=nfpm"`
 	Provides   []string       `yaml:"provides,omitempty" json:"provides,omitempty" jsonschema:"title=provides directive,example=nfpm"`
+	Predepends []string       `yaml:"predepends,omitempty" json:"predepends,omitempty" jsonschema:"title=predepends directive,example=nfpm"`
```

Comment:
since this is only for debs, it should probably go into the specific deb options struct. (line 385)